router/create.js

```diff
@@ -0,0 +1,51 @@
+import microCors from 'micro-cors';
+
+const isHappyMiddleware = (middlewareHandler) => middlewareHandler.length === 3; // (request, response, next)
+const isSadMiddleware = (middlewareHandler) => middlewareHandler.length === 4; // (error, request, response, next)
+
+function createRouter(...middlewareHandlers) {
+  const happyMiddlewares = middlewareHandlers.filter(isHappyMiddleware);
+  const sadMiddlewares = middlewareHandlers.filter(isSadMiddleware);
+
+  return function use(handler, options) {
```

Comment:
O que acha de mover o `...middlewareHandlers` pra cá? Assim eu consigo definir os middleware na hora que for definir as rotas, muito parecido com a API do express
```
useRouter(
  middleware1,
  middleware2,
  action
)
```

Dessa forma, para termos o errorHandler global, seria só fazer um `push` no sadMiddlewares colocando o error handler padrão como último a ser executado